lib/platform/github/gh-graphql-wrapper.ts

```diff
@@ -0,0 +1,71 @@
+import { api } from './gh-got-wrapper';
+import { logger } from '../../logger';
+
+const gqlOpts = {
+  headers: {
+    accept: 'application/vnd.github.merge-info-preview+json',
+  },
+  json: false,
+};
+
+interface GithubGraphqlResponse {
+  data?: {
+    repository?: any;
+  };
+  errors?: any;
+}
+
+async function gqlGet(query: string): Promise<GithubGraphqlResponse> {
+  try {
+    const body = JSON.stringify({ query });
+    const options = { ...gqlOpts, body };
+    const res = await api.post('graphql', options);
+    return JSON.parse(res.body);
+  } catch (err) {
+    logger.warn({ query, err }, 'graphql.get error');
+    throw new Error('platform-error');
```

Comment:
Is this behaviour exactly the same as today? i.e. if error thrown then we throw platform-error?

Reply:
Thanks, changed to `platform-failure`